app.rb

```diff
@@ -0,0 +1,24 @@
+require 'sinatra'
+require_relative 'lib/tax_tribunal'
+require 'pry'
+
+module TaxTribunal
+  class Downloader < Sinatra::Base
```

Comment:
Might be worth moving the file to `lib/tax_tribunal/downloader.rb` to match the class name.